src/Microsoft.Azure.SignalR/ServiceOptions.cs

```diff
@@ -76,5 +76,10 @@ public class ServiceOptions : IServiceEndpointOptions
         /// Gets or sets the proxy used when ServiceEndpoint will attempt to connect to Azure SignalR.
         /// </summary>
         public IWebProxy Proxy { get; set; }
+
+        /// <summary>
+        /// Specifies the timeout of scaling endpoints in running app (in minutes). 
+        /// </summary>
+        public TimeSpan ServiceScaleTimeout { get; set; } = TimeSpan.FromMinutes(Constants.DefaultScaleTimeoutInMinutes);

```

Comment:
Maybe we should leave this option internal since the feature the PR is only the very first part of the feature?